swap/src/monero/wallet.rs

```diff
@@ -195,8 +195,8 @@ impl Wallet {
             if proof.confirmations > confirmations.load(Ordering::SeqCst) {
                 confirmations.store(proof.confirmations, Ordering::SeqCst);
                 info!(
-                    "Monero lock tx received {} out of {} confirmations",
-                    proof.confirmations, expected_confirmations
+                    "Received {} out of {} confirmations for Monero lock tx {}",
```

Comment:
Confirmations is not really something that one "receive"s.

Can we rephrase this somehow?
I like that the `{} out of {}` is relatively early in the message because it is the most relevant information.

Reply:
Maybe:

```suggestion
                    "Monero lock tx has {} out of {} confirmations. TXID = {}"
```

We could establish this pattern in several places. Maybe even use tracing's value capture for it?